books/RayTracingTheRestOfYourLife.html

```diff
@@ -1425,32 +1426,34 @@
 strength in the RGB color space, colors can also be represented by how excited each set of cones is
 in the _LMS color space_ (long, medium, short).
 
+Scattering
+-----------
 If the light does scatter, it will have a directional distribution that we can describe as a PDF
 over solid angle. I will refer to this as its _scattering PDF_: $\mathit{pScatter}()$. The
-scattering PDF will vary with the outgoing direction: $\mathit{pScatter}(\omega_o)$. The scattering
-PDF can also vary with _incident direction_: $\mathit{pScatter}(\omega_i, \omega_o)$. You can see
-this varying with incident direction when you look at reflections off a road -- they become
-mirror-like as your viewing angle (incident angle) approaches grazing. Lastly, the scattering PDF
-can also depend on the scattering position: $\mathit{pScatter}(\mathbf{x}, \omega_i, \omega_o)$. The
+scattering PDF will vary with outgoing direction: $\mathit{pScatter}(\omega_o)$. The scattering PDF
+can also vary with _incident direction_: $\mathit{pScatter}(\omega_i, \omega_o)$. You can see this
+varying with incident direction when you look at reflections off a road -- they become mirror-like
+as your viewing angle (incident angle) approaches grazing. The scattering PDF can vary with the
+wavelength of the light: $\mathit{pScatter}(\omega_i, \omega_o, \lambda)$. A good example
+of this is a prism refracting white light into a rainbow. Lastly, the scattering PDF can also depend
+on the scattering position: $\mathit{pScatter}(\mathbf{x}, \omega_i, \omega_o, \lambda)$. The
 $\mathbf{x}$ is just a vector representing the scattering position: $\mathbf{x} = (x, y, z)$. The
```

Comment:
Just a vector
or
Just a point
?

Reply:
iiiiiiiiiiii

it's both. Ugh.
It's a mathematical vector, which is subtly different from the vector we've been using

Reply:
Your choice. I just thought it was worth the callout.